src/features/DisplayControls/Treatments/styles/Treatments.scss

```diff
@@ -0,0 +1,3 @@
+.no-treatments {
+  padding-top: 0.625rem;
```

Comment:
can we use percentage instead of rem?

Reply:
in case of padding and margin px units are also better depending on the use case